src/api/index.js

```diff
@@ -37,7 +36,7 @@ export const useGetPlaybookPreview = (data) => {
         const previewData = dataTransformer(data);
         const playbookPreview =
           previewData &&
-          (await axios.post(`${V1_API_BASE}/states/preview`, {
+          (await axios.post(`${CONNECTOR_API_BASE}/states/preview`, {
```

Comment:
@mkholjuraev there is no `v2/states/preview` endpoint in v2 API.
You can use this new endpoint [/playbooks](https://github.com/RedHatInsights/config-manager/blob/master/internal/http/v2/openapi.json#L204) of v2 with `profile_id` param i.e `/v2/playbooks?profile_id=%v`

Reply:
@r14chandra aha, thank you for pointing this out. Can you give more context on what are the differences between the old endpoint and the new one? Unfortunately, the OpenApi docs do not really show this.

Reply:
Both are same. The` v2` API was introduced to resolve race condition in `v1` API - [ESSNTL-3301](https://issues.redhat.com/browse/ESSNTL-3301). Let me know if this helps.

Reply:
As of now, the UI gets a state map from the Model. But now we need to pass `profile_id` of the current config state. I think here we can use `v2/profiles/current` to get the currently applied config and from there we can get the profile ID. What do you think?

Reply:
Yes, that should be doable. Thank you!

Reply:
@r14chandra now, everything should be fine. PTAL!